minicluster/src/main/java/org/apache/accumulo/minicluster/impl/MiniAccumuloClusterImpl.java

```diff
@@ -715,7 +716,11 @@ public String getZooKeepers() {
 
   @Override
   public ServerContext getServerContext() {
-    return new ServerContext(getClientInfo());
+    try {
+      return new ServerContext(config.getAccumuloSiteFile().toURI().toURL());
```

Comment:
This should be created once for mini... it shouldn't really ever fail, since it's always providing its own configuration file... that it created.